src/winapp-CLI/WinApp.Cli/Services/BuildToolsService.cs

```diff
@@ -347,4 +397,11 @@ public InvalidBuildToolException(int processId, string stdout, string stderr, st
 
     [GeneratedRegex(@"^\d+\.\d+\.\d+\.\d+$")]
     private static partial Regex VersionFolderRegex();
+
+    // Recognize lines that look like error information.
+    // Example strings we want to match:
+    //   error:
+    //   general error c1010070:
+    [GeneratedRegex(@"error\s+[a-z]?\d+:|error:", RegexOptions.IgnoreCase)]
+    private static partial Regex ErrorLineRegex();
```

Comment:
Should we filter the lines like this? I'm wondering if we just output everything if the return code is not zero. Wouldn't that be better? Maybe the tool being called outputs in multiple lines and the actual error would not be printed?